### DISCUSSION TITLE
Full name

### DISCUSSION THREAD

**Discussion by Chaose3** [community/community#146289] [GitHub Education] [answered]
### Hi everyone,

Hi everyone,
I am trying to get a educational github account and recieved an email saying that I must have my full name on my github profile. I have a very unique last name that can be used to find where I live and do not want people to know such things about me.

**Comment by Stansbury-C** [comment, marked as answer]
Can I check if this is asking for your full name in the billing details section of your account? I believe this is where I had to provide my full name however my profile name has always been Christopher and not my Full name.

If this is the case head to settings, billing and update it there.
